pkg/watchdogd/watchdogd.go

```diff
@@ -190,7 +190,7 @@ func (d *Daemon) DisarmWatchdog() rune {
 		log.Printf("Failed to disarm watchdog: %v", err)
 		return OpResultError
 	}
-	log.Println("Watchdog disarmed.")
+	log.Println("Watchdog disarming request went through (Watchdog will not be disabled if CONFIG_WATCHDOG_NOWAYOUT is enabled).")
```

Comment:
I'm somewhat incredulous that there seems to be no reliable way to find out if nowayout is set. But there doesn't seem to be. It's an obvious place for a sysctl but ... geez. Who writes something like that?

Reply:
hmm, we can find it out by looking for a `Y` in `/sys/module/sbsa_gwdt/parameters/nowayout`.

I can see it work on a arm64. not clear if that is reliable. Likely some driver change to expose it as a common interface. 

But take a step back, why would you want to check if it is enabled ? try not to actually disable it ? --- leap and ask for forgiveness seems fine

Reply:
because it basically says "who knows?" -- it's a pretty useless message at this point.

Reply:
User knows.

User already knows if their kernel has `CONFIG_WATCHDOG_NOWAYOUT` enabled or not. If enabled, it won't be stopped regardless. Otherwise, the log message means watchdog is disarmed. Agree ?

ideally, driver could return an error instead of just eating the request. Currently, It chooses to print `[  209.655385] watchdog: watchdog0: watchdog did not stop!` .

Reply:
but happy to take suggestions ! if there is better way to handle this